CHANGELOG.md

```diff
@@ -0,0 +1,160 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.2.0] - 2020-02-05
+### Added
+- Analyzers can now return a boolean or **a list of matched results**
+- Actions now get passed a list of matched results by the analyzer
+- New Analyzer: PasteTitleAnalyzer - Analyzer to match Paste titles via regex
+- New Analyzer: IPv4AddressAnalyzer - Match IPv4 addresses via regex
+- Subclasses of RegexAnalyzer now got a method `def verify(results)` that can be overwritten to filter matches so you only return valid results
+- EmailPasswordPairAnalyzer has now an optional parameter `min_amount` to specify how many pairs must be found to actually match
+- Base64Analyzer got an optional parameter `min_len` to specify how long a detected string must be at least to actually match
+- Logical operators for analyzers - you can now connect multiple analyzers with logical operators to specify more precisely when a paste should match (aed2dbf615e7be4586747f5ea6f437663b422f57)
+### Changed
+- Analyzers can now return a boolean or a list of matched results
+- Actions now get passed a list of matched results by the analyzer and can
+- IBANAnalyzer will now filter out wrong IBANs and return a list of validated IBANs if the `validate` parameter is set to `True`
+### Fixed
+- Using non-capturing groups in regex for various analyzers. This is done so that the analyzer can return a matched string and at the same time it fixed some issues with analyzers not matching properly
+
+## [1.1.0] - 2019-11-11
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/d-Rickyy-b/pastepwn/pullRequest?prid=4945865)